File: spring-reactive-02/learn-java/src/main/java/boki/learnjava/v1_newsequence/d_range/RangeEx1.java
package boki.learnjava.v1_newsequence.d_range;

import boki.learnjava.utils.Logger;
import reactor.core.publisher.Flux;

/**
 * range() 사용 예제
 * 첫번째 파라미터(start)부터 두번째 파라미터(count)만큼 1씩 증가한 연속된 정수를 emit
 */
public class RangeEx1 {
    public static void main(String[] args) {
        Flux
            .range(5, 10)
            .subscribe(Logger::onNext);
    }
}

File: spring-reactive-02/learn-kt/src/main/kotlin/boki/learnkt/v1_newsequence/range/RangeEx1.kt
package boki.learnkt.v1_newsequence.range

import boki.learnkt.util.Logger
import reactor.core.publisher.Flux

/**
 * range() 사용 예제
 * 첫번째 파라미터(start)부터 두번째 파라미터(count)만큼 1씩 증가한 연속된 정수를 emit
 */
fun main() {
    Flux
        .range(5, 10)
        .subscribe { Logger.onNext(it) }
}